src/index.spec.js

```diff
@@ -246,16 +262,16 @@ describe('desciple-core-api', () => {
 
       let verifiedAttestor = await discipl.verify('agree', claimlink, [attestorSsid])
 
-      expect(verifyStub.callCount).to.equal(4)
+      expect(verifyStub.callCount).to.equal(3)
       expect(verifyStub.args[0]).to.deep.equal([{ did: 'did:discipl:mock:attestor', connector: stubConnector, pubkey: 'attestor' }, { agree: claimlink }])
       expect(verifyStub.args[1]).to.deep.equal([{ did: 'did:discipl:mock:attestor', connector: stubConnector, pubkey: 'attestor' }, { revoke: attestationlink }])
       expect(verifyStub.args[2]).to.deep.equal([{ did: 'did:discipl:mock:attestor', connector: stubConnector, pubkey: 'attestor' }, { revoke: attestationrevocationlink }])
-      expect(verifyStub.args[3]).to.deep.equal([{ did: 'did:discipl:mock:attestor', connector: stubConnector, pubkey: 'attestor' }, { revoke: attestationlink }])
+      // expect(verifyStub.args[3]).to.deep.equal([{ did: 'did:discipl:mock:attestor', connector: stubConnector, pubkey: 'attestor' }, { revoke: attestationlink }])
 
-      expect(getSsidOfClaimStub.calledOnce).to.equal(true)
-      expect(getSsidOfClaimStub.args[0]).to.deep.equal(['attestationRef'])
+      expect(getSsidOfClaimStub.calledOnce).to.equal(false)
```

Comment:
If a stub is not called, I propose we remove the entire stub (as calling this function will then fail the test by virtue of the stub not existing)